### PR TITLE
[Elu x86] Finish intrinsic with elempack merged

### PR DESCRIPTION
**Pull request by LRY89757**
* 实现了Elu的x86平台优化
* 添加了elu_sse/avx/avx512在x86_activation的实现
* 添加了test_sample追求覆盖率.

### CODE REVIEW

File: src/layer/x86/elu_x86.cpp [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+            __m256 _p = _mm256_loadu_ps(ptr);
+            _mm256_storeu_ps(ptr, elu_avx(_p, _alpha256));
+
+            ptr += 8;
+        }
+#endif // __AVX__
+        for (; i + 3 < size; i += 4)
+        {
+            __m128 _p = _mm_loadu_ps(ptr);
+            _mm_storeu_ps(ptr, elu_sse(_p, _alpha128));
```
Comment (nihui) [inline src/layer/x86/elu_x86.cpp:69]: sse2 load/store shall be always aligned

File: src/layer/x86/elu_x86.cpp [hunk 2]
```diff
@@ -0,0 +1,85 @@
+// Tencent is pleased to support the open source community by making ncnn available.
+//
+// Copyright (C) 2017 THL A29 Limited, a Tencent company. All rights reserved.
```
Comment (nihui) [inline src/layer/x86/elu_x86.cpp:3]: 2022

File: src/layer/x86/elu_x86.h [hunk 3]
```diff
@@ -0,0 +1,32 @@
+// Tencent is pleased to support the open source community by making ncnn available.
+//
+// Copyright (C) 2017 THL A29 Limited, a Tencent company. All rights reserved.
```
Comment (nihui) [inline src/layer/x86/elu_x86.h:3]: 2022

**Review by nihui — CHANGES_REQUESTED** [review]

File: src/layer/x86/elu_x86.h (hunk 3 above)
Comment (LRY89757) [reply src/layer/x86/elu_x86.h:3]: Get it!

Comment (LRY89757) [reply src/layer/x86/elu_x86.h:3]: I will check the other prs later.

**Comment by nihui** [comment]
Thanks for your contribution !

**Pull request merged**